components/suggestion/generic_user_provider.tsx

```diff
@@ -24,7 +25,7 @@ export type ProviderResults = {
     component?: React.ReactNode;
 }
 
-class UserSuggestion extends Suggestion {
+class UserSuggestion extends Suggestion<ProviderResults> {
```

Comment:
Same thing here. It might be enough to use the default `Item` type in this case though